worldguard-core/src/main/java/com/sk89q/worldguard/domains/DefaultDomain.java

```diff
@@ -257,7 +326,7 @@ public boolean contains(String playerName) {
 
     @Override
     public int size() {
-        return groupDomain.size() + playerDomain.size();
+        return groupDomain.size() + playerDomain.size() + customDomains.size();
```

Comment:
Shouldn't this call `size()` for each element in `customDomains`?

Reply:
I'm not completly sure how to handle this. Technically an empty custom domain "is there" and could change stuff. So I guess this implementation should be fine.
We can change it later if there are any issues.